### PR TITLE
Create hbcd_pe.gns3a

### LINKED ISSUE #919: HBCD PE
Hello,

I'm trying HBCD PE https://www.hirensbootcd.org/ in GNS3 with the following appliance.

```
{
    "appliance_id": "ac98ab6f-7966-444b-842f-9507c965b8b7",
    "name": "HBCD-PE",
    "category": "guest",
    "description": "Hiren’s BootCD PE (Preinstallation Environment) is a restored edition of Hiren’s BootCD based on Windows 11 PE x64. ",
    "vendor_name": "hirensbootcd.org",
    "vendor_url": "https://www.hirensbootcd.org/",
    "documentation_url": "https://www.hirensbootcd.org/howtos/",
    "product_name": "Hiren’s BootCD PE",
    "product_url": "https://www.hirensbootcd.org/",
    "registry_version": 4,
    "status": "stable",
    "maintainer": "GNS3 Team",
    "maintainer_email": "developers@gns3.net",
    "usage": "",
    "symbol": "cd.svg",
    "qemu": {
        "adapter_type": "e1000",
        "adapters": 1,
        "ram": 4096,
        "hda_disk_interface": "sata",
        "arch": "x86_64",
        "console_type": "vnc",
        "boot_priority": "c",
        "kvm": "require"
    },
    "images": [
        {
            "filename": "HBCD_PE_x64.iso",
            "version": "1.0.8",
            "md5sum": "45baab64b088431bdf3370292e9a74b0",
            "filesize": 3291686912,
            "download_url": "https://www.hirensbootcd.org/download/",
            "direct_download_url": "https://www.hirensbootcd.org/files/HBCD_PE_x64.iso"
        },
        {
            "filename": "empty30G.qcow2",
            "version": "1.0",
            "md5sum": "3411a599e822f2ac6be560a26405821a",
            "filesize": 197120,
            "download_url": "https://sourceforge.net/projects/gns-3/files/Empty%20Qemu%20disk/",
            "direct_download_url": "https://sourceforge.net/projects/gns-3/files/Empty%20Qemu%20disk/empty30G.qcow2/download"
        }
    ],
    "versions": [
        {
            "name": "1.0.8",
            "images": {
                "hda_disk_image": "empty30G.qcow2",
                "cdrom_image": "HBCD_PE_x64.iso"
            }
        }
    ]
}
```

But I'm getting

image

File: \windows\system32\drivers\iaVROC.sys

A critical system driver is missing

Any idea?

### PR DESCRIPTION
**Pull request by scls19fr**
Closes #919

Before submitting a pull request, please check the following.

---
When updating an **existing** appliance:
- [ ] The new version is on top.
- [ ] The filenames in the "images" section are unique, to avoid appliances / version overwriting each other.
- [ ] If you forked the repo, running check.py doesn't drop any errors for the updated file.
---
When creating a **new** appliance:
- It's tested locally, i.e.
  - [ ] You dragged an instance into a project on your box, got it installed (if necessary), and did some basic network checks (ping, UI reachable, etc.).
  - [ ] GNS3 VM can run it without any tweaks.
  - [ ] The device is in the right category: router, switch, guest (hosts), firewall
  - [ ] You filled in as much info as possible (checks the schemas and other appliance files for some guidance).
- [ ] When adding a container: it builds on Docker Hub and can be pulled.
- [ ] The filenames in the "images" section are unique (to avoid appliances and/or versions overwriting each other).
- [ ] If you forked the repo, running check.py doesn't drop any errors for the new file.
- [ ] *Optional: a symbol has been created for the new appliance.*

### CODE REVIEW

File: appliances/hbcd_pe.gns3a [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "vendor_name": "hirensbootcd.org",
+    "vendor_url": "https://www.hirensbootcd.org/",
+    "documentation_url": "https://www.hirensbootcd.org/howtos/",
+    "product_name": "Hiren’s BootCD PE",
+    "product_url": "https://www.hirensbootcd.org/",
+    "registry_version": 4,
+    "status": "stable",
+    "maintainer": "GNS3 Team",
+    "maintainer_email": "developers@gns3.net",
+    "usage": "",
```
Comment (grossmj) [inline appliances/hbcd_pe.gns3a:15]: I think you can remove the usage field if it is empty.

File: appliances/hbcd_pe.gns3a [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    "symbol": "cd.svg",
```
Comment (grossmj) [inline appliances/hbcd_pe.gns3a:16]: Is this a builtin-in symbol or do you want to provide one?

Comment (scls19fr) [reply appliances/hbcd_pe.gns3a]: Sorry my mistake.
I only have 
https://commons.wikimedia.org/wiki/File:CD_icon.svg
https://fr.m.wikipedia.org/wiki/Fichier:CD_icon_test.svg
https://en.m.wikipedia.org/wiki/File:CD_icon.svg
Not sure what is the most appropriate licence to start with
I think ideally we should have a computer showing a CD (either on the screen or next to keyboard).
I'm sorry but I don't have artistic drawing skills ;-)

**Comment by grossmj** [comment]
Thanks again for the contribution 👍

**Pull request merged**